docs/_includes/tk-photoshopcc

```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 2024 (v24.1.1)
```

Comment:
Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

According to https://en.wikipedia.org/wiki/Adobe_Photoshop#2021_(version_22), it should be 2021 (v22)
?